src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -85,6 +92,20 @@ const UserInformationContent: React.FunctionComponent<PropsType> = ({
         onClick={logout}
         type="button"
       />
+
+      <label
+        htmlFor={"chk-kidsMode"}
+        className={`checkbox ${profile?.kidsMode && "checkbox-checked"}`}
+      >
+        Kids Mode
```

Comment:
Wonder if we should have something extra here to describe it, like:

`Kids Mode (do not show venues with 18+ content)`